tool/resources/org/antlr/v4/tool/templates/codegen/Java/Java.stg

```diff
@@ -898,9 +898,16 @@ public class <lexer.name> extends <superClass; null="Lexer"> {
 		<lexer.tokens:{k | <k>=<lexer.tokens.(k)>}; separator=", ", wrap, anchor>;
 	<if(lexer.channels)>
 	public static final int
-		<lexer.channels:{k | <k>=<lexer.channels.(k)>}; separator=", ", wrap, anchor>;
+		<lexer.channels:{c | <c>=<lexer.channels.(c)>}; separator=", ", wrap, anchor>;
 	<endif>
-	<rest(lexer.modes):{m| public static final int <m> = <i>;}; separator="\n">
+	<if(rest(lexer.modes))>
+	public static final int
+		<rest(lexer.modes):{m | <m>=<i>}; separator=", ", wrap, anchor>;
```

Comment:
I changed mode format from
```Java
public static final int MODE1 = 1;
public static final int MODE2 = 2;
```

to the tokens format:

```Java
public static final int
    MODE1 = 1, MODE2 = 2;
```

Channels also have the same format. This is more convenient. Are you agree with me?